vignettes/protein_structure_workflow.Rmd

```diff
@@ -0,0 +1,457 @@
+
+# Mapping of amino acids, peptides or regions on protein structures and predictions
+
+Contact maps provide quantitative information about the distances of residues to each other. We can use these to perform further high throughput analysis. In contrast to the rather complex contact map plots it is easier to look at the actual protein structures and the respective peptide positions to get a better overview of the proximity of peptides to each other and in relation to interesting sites. While this analysis is more subjective, it can make certain relationships in the data more obvious.
+
+For the mapping of peptides, protein regions or amino acids on PDB structures or AlphaFold predictions, we provide the `map_peptide_on_structure()` function. Mapping is accomplished based on the replacement of B-factor information in the structure file with peptide specific values. When the structure is coloured by B-factor in PyMOL or ChimeraX, peptides will get highlighted. This function can on the one hand just display categorical information such as if the peptide is significant, present or undetected. On the other hand, continuous information can be displayed such as a score associated with each individual amino acid.
+
+The mapping is performed based on positions and not peptide sequence matching. If you want to perform the mapping onto a PDB structure, you should run the `find_peptide_in_structure()` beforehand to obtain structure specific positions. If you want to map onto an AlphaFold prediction it is not necessary to carry out this step. However, the output of the `find_peptide_in_structure()` function will contain all necessary information to perform also AlphaFold predictions in case there is no available structure file. 
+
+For this function it is important to provide the author defined positions and chain names which are marked with the `auth` prefix. This is the case because the function performs structural mapping based on this naming convention. 
+
+_Note: If you perform mapping onto an AlphaFold prediction the content of the `pdb_id` provided column should be NA._
+
+_Note: If you want to map peptides to an unpublished or simply downloaded structure that you have the .pdb or .cif file of, you can provide the path to that structure to the `structure_file` argument. In this case you do not need to provide the `pdb_id` argument. It is very important that your input data frame `peptide_data` only contains information for the protein in the provided structure file and not for other unrelated proteins, otherwise mapping is not performed correctly._
```

Comment:
```suggestion
_Note: If you want to map peptides to an unpublished or downloaded structure that you have the .pdb or .cif file of, you can provide the path to that structure to the `structure_file` argument. In this case you do not need to provide the `pdb_id` argument. It is very important that your input data frame `peptide_data` only contains information for the protein in the provided structure file and not for other unrelated proteins, otherwise mapping is not performed correctly._
```